### DISCUSSION TITLE
After updating to Livewire 2.6.0 start getting Entangle Error

### DISCUSSION THREAD

**Discussion by yabasha** [livewire/livewire#3750] [7. Bugs (archived)] [answered]
Hi guys,

I was using Livewire 2.5.5 and everything was working fine but after I updated livewire to v2.6.0 `Livewire Entangle Error` started appearing in the browser console.

![image](https://user-images.githubusercontent.com/932321/133423428-23f8e802-557b-4a8e-814e-c81ffa493b7a.png)

**Comment by yabasha** [comment]
If I use trix component the error start appearing but when I print the property it works.

Trix editor code:
```blade
<div class="mt-1" x-data="{
    value: @entangle($attributes->wire('model')),
    isFocused(){ return document.activeElement !== this.$refs.trix },
    setValue(){ this.$refs.trix.editor.loadHTML(this.value) },
}" x-init="$nextTick(() => {setValue(); $watch('value', () => isFocused() && setValue()); })"
    x-on:trix-change="value = $event.target.value" {{ $attributes->whereDoesntStartWith('wire:model') }} wire:ignore>
    <trix-editor x-ref="trix"
        class="shadow-sm focus:ring-rose-500 focus:border-rose-500 block w-full sm:text-sm border-gray-300 rounded-md resize-none">
    </trix-editor>
</div>

```

**Reply by yabasha** [reply]
But when I am using the following textare blade component it works fine:
```blade
<div class="mt-1">
    <textarea {{ $attributes->merge(['class' => 'shadow-sm focus:ring-rose-500 focus:border-rose-500 block w-full sm:text-sm border-gray-300 rounded-md']) }}"></textarea>
</div>
```

the error message in `@entangle` , can anybody help in this issue.

**Comment by joshhanley** [comment]
@yabasha can you share your component class and view?

**Reply by yabasha** [reply]
I renamed `editing.body` to `editing.content` but it didn't work:

Livewire view: https://pastebin.com/JL9x4X3R

Livewire component class: https://pastebin.com/ZgTdkVHX

**Reply by joshhanley** [reply]
@yabasha thanks for sharing! Hmm all looks ok, I'm not sure why `@entangle` can't find the property. I will keep digging.

**Comment by andreas-freitag** [comment]
I get the same error, even though I am not using Trix but Flatpickr and TinyMCE.

If I change the property to be directly on the Livewire Component and not an Eloquent models attribute, it is working again. So, I suspect a problem that was caused by the new deep model binding. Maybe that helps to point the search in the right direction.

**Reply by joshhanley** [reply]
@andreas-freitag ok thanks for the info, `@entangle` is working fine for me with models, so I wonder what is going on. 

Do you a component class and view you can share so I can compare it with Yabasha's one above?

**Comment by joshhanley** [comment, marked as answer]
@yabasha and @andreas-freitag I managed to find the issue thanks to both of your help 🙂

Turns out, if you make a new model (but not persist it) then the properties don't exist yet on the object. So when I built the new deep model binding stuff, I was only filtering by properties that exist.

I have fixed that and tagged a new release 2.6.4.

Could you please test it and let me know if it has fixed the issue for you?

Thanks!

**Reply by andreas-freitag** [reply]
@joshhanley 
Thanks for the quick response. Really appreciate it! 

And it was indeed a not yet persisted model in my case. 
Your fix solves the issue for me, thanks again !

**Reply by yabasha** [reply]
Thanx @jonneroelofs for your support, this fixes my issue too.

**Reply by joshhanley** [reply]
Thank you both for testing, glad it worked 🙂
